whisper.cpp

```diff
@@ -3523,6 +3543,11 @@ static void whisper_process_logits(
         logits[vocab.token_sot]  = -INFINITY;
         logits[vocab.token_solm] = -INFINITY;
```

Comment:
For the same reason as above, the suppression of `token_solm` should also be moved inside the `params.tdrz_enable` conditional.

Or since it is an unused token, the other option is that you can rename `token_solm` to `token_nospeech`.